tests/integration/distribution.go

```diff
@@ -338,8 +340,156 @@ func (s *CCVTestSuite) TestEndBlockRD() {
 	}
 }
 
+// TestSendRewardsToProvider is effectively a unit test for TestSendRewardsToProvider(),
```

Comment:
Maybe "a unit test for `SendRewardsToProvider`" instead of "a unit test for TestSendRewardsToProvider(),"?